src/backend/CassandraBackend.cpp

```diff
@@ -1558,12 +1679,28 @@ CassandraBackend::open(bool readOnly)
             continue;
 
         query.str("");
-        query << "INSERT INTO " << tablePrefix << "issuer_nf_tokens"
-              << " (issuer,token_id)"
-              << " VALUES (?,?)";
+        query << "INSERT INTO " << tablePrefix << "issuer_nf_tokens_v2"
```

Comment:
correct. this is why, unless you can call the BackendInterface constructor at the end of the CassandraBackend constructor, I think the best place to put the call to `runMigrations` is in the factory. I don't like the former approach even if it is possible, because it creates a trap for future developers of new backends who have to _remember_ to call the BackendInterface constructor in their constructor.

Reply:
No no no :D just leave migrations to the actual backend, not the interface.